src/main/java/life/mosu/mosuserver/domain/auth/security/RefreshTokenRedisEntity.java

```diff
@@ -0,0 +1,31 @@
+package life.mosu.mosuserver.domain.auth.security;
+
+import org.springframework.data.annotation.Id;
+import lombok.Getter;
+import org.springframework.data.redis.core.RedisHash;
+import org.springframework.data.redis.core.TimeToLive;
+import org.springframework.data.redis.core.index.Indexed;
+
+import java.util.concurrent.TimeUnit;
+
+@Getter
+@RedisHash(value = "token_verification")
+public class RefreshTokenRedisEntity {
+
+    @Id
+    private Long userId;
+
+    @Indexed
+    private String refreshToken;
+
+    @TimeToLive(unit = TimeUnit.MILLISECONDS)
+    private Long expiration;
+
+    public static RefreshTokenRedisEntity from(final RefreshToken token) {
+        final RefreshTokenRedisEntity entity = new RefreshTokenRedisEntity();
+        entity.userId = token.userId();
+        entity.refreshToken = token.refreshToken();
+        entity.expiration = token.expiration();
+        return entity;
+    }
+}
```

Comment:
```suggestion
@Getter
@RedisHash(value = "token_verification")
@AllArgsConstructor
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class RefreshTokenRedisEntity {

    @Id
    private Long userId;

    @Indexed
    private String refreshToken;

    @TimeToLive(unit = TimeUnit.MILLISECONDS)
    private Long expiration;
    
    public static RefreshTokenRedisEntity from(final RefreshToken token) {
        return new RefreshTokenRedisEntity(token.userId(), token.refreshToken(), token.expiration());
    }
}
````

일단 코드의 간결성 측면으로 볼 때, 수동 주입하는 형태보다는 이러한 형태는 어떤가요??